docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -118,6 +128,9 @@ values. Typical usage does not require you to configure them.
       - Name of the catalog to which ``SELECT`` queries are redirected when a
         Hive table is detected.
       -
+    * - ``delta.checkpoint-row-statistics-writing.enabled``
+      - Enables writing row statistics to checkpoint files.
```

Comment:
I don't know, from how they are used it is not clear to me. 
I only know they ARE NOT used for query planning ;) (https://github.com/trinodb/trino/blob/bff3e64ddd392b85ed0bf27e878c42dba1395de0/plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointEntryIterator.java#L492)

If you know please propose something.

Reply:
Original commit comment:

> Row statistics may create a large memory footprint. Add a config option to disable them in case this occurs.

@alexjo2144 could you please give some extra context here.

Reply:
We should eventually remove this property and respect the table property for it: `delta.checkpoint.writeStatsAsStruct` 

@homar can you create an issue if there isn't one already?

Reply:
@alexjo2144 I added a comment to https://github.com/trinodb/trino/issues/12636 and I think now it covers what we need here. So I think this can be left in this form temporarily